File: day002/day002.c
#include <stdio.h>

int main() {
	int x, y;
	printf("Bir sayi giriniz: ");
	scanf("%d", &x);
	printf("Bir sayi daha giriniz: ");
	scanf("%d", &y);
	printf("Girdiginiz sayilarin toplami: %d", (x+y));
	return 0;
}

File: day002/Day002.cs
using System;

	class Day002
{
	static void Main()
	{
	int x, y;
	Console.Write("Bir sayi giriniz: ");
	x = Convert.ToInt32(Console.ReadLine());
	Console.Write("Bir sayi daha giriniz: ");
	y = Convert.ToInt32(Console.ReadLine());
	Console.WriteLine("Girdiginiz sayilarin toplami: " + (x+y)); 
	}
}
